finished styling and refactor Answers component

sunkim0330 opened the pull request:
@AlexArms @kate-darby @chrimack Hi yall, I just finished styling and refactor Answers to look cleaner.
If there is any stylings yall don't like, just let me know :) I'll fix it.

sunkim0330 committed:
Merge branch 'helpful' of https://github.com/FEC-Zion/catwalk into helpfulStyling

sunkim0330 committed:
almost done. working on buttons

sunkim0330 committed:
done with styling...for now

sunkim0330 committed:
merge conflict in test branch

sunkim0330 committed:
changed background color for modal

sunkim0330 committed:
this is a test branch. didn't mean to change anything here

sunkim0330 committed:
merged, and had merge conflict in App.jsx

sunkim0330 committed:
DONE with stylings!

sunkim0330 committed:
pull my finished branch on testing branch to test colorTheme

sunkim0330 committed:
Merge branch 'test' of https://github.com/FEC-Zion/catwalk into colorThemeTest

sunkim0330 committed:
colorTheme implemented

sunkim0330 committed:
implement dark mode successfully

sunkim0330 committed:
finally done with styling (hopefully)

sunkim0330 committed:
refactor Answers.jsx

sunkim0330 committed:
Merge branch 'development' into colorThemeTest

AlexArms commented on client/src/Shared/Helpful.jsx:
```diff
@@ -28,6 +28,20 @@ const Helpful = ({ origin, id, helpCount }) => {
+      styles['border-right'] = '1px solid black;';
+    }
+    return styles;
+  });
```
This block needs to be removed or used, I think removed because it looks like a duplicate of something above.

sunkim0330 replied:
I just fixed it!

sunkim0330 committed:
removed duplicates

I removed duplicate from Helpful component. I thought I accepted incoming change, not both during resolving conflict.
